rolling/distribution.yaml

```diff
@@ -588,6 +588,24 @@ repositories:
       url: https://github.com/ros2/cartographer_ros.git
       version: ros2
     status: maintained
+  cascade_lifecycle:
+    doc:
+      type: git
+      url: https://github.com/fmrico/cascade_lifecycle.git
+      version: rolling-devel
+    release:
+      packages:
+      - cascade_lifecycle_msgs
+      - rclcpp_cascade_lifecycle
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/fmrico/cascade_lifecycle-release.git
```

Comment:
Our best practices here are to use an https://github.com/ros2-gbp repository for releases into Rolling. That way, we'll be able to automatically release this package from Rolling into the next ROS distribution (Iron). If you'd like to do this, then please see https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md on how to create one.

This is particularly important right now as we're preparing for the iron fork. With a ros2-gbp repo we will be able to make that fork automatically.

Reply:
Hi @tfoote, sorry for this. Of course, I can do it. 

I have just opened https://github.com/ros2-gbp/ros2-gbp-github-org/issues/168 . I suppose I have to wait for the repository import to ros2-gbp

Thanks!!!

Reply:
Thanks for requesting that. I've pinged nuclearsandwich about it.